source/designers-guide/smarty-plugins/index.md

```diff
@@ -0,0 +1,213 @@
+---
+layout: default
+title: Smarty Plugins
+github_link: designers-guide/smarty-plugins/index.md
+indexed: true
```

Comment:
Please add the info for the menu structure to also add a menu link.

For example:
```
group: Frontend Guides
subgroup: General Resources
menu_title: Smarty Plugins
menu_order: 55
```

Reply:
added